Model/ProductProtection.php

```diff
@@ -374,15 +400,15 @@ private function upsert(
                     );
                 }
                 $item = $this->itemFactory->create();
+
+                if (isset($quantity)) {
```

Comment:
Do we need this check if we have line [391](https://github.com/helloextend/magento-extend-integration/pull/108/files#diff-2045ad538de09c37b82125e3ab9b626ff507083f689158a5d24b42b4f67ef0faR391)?

Reply:
good catch, this has been removed